apps/vmq_server/src/vmq_ranch_sup.erl

```diff
@@ -27,6 +27,24 @@
 %%% API functions
 %%%===================================================================
 
+active_connections(Ip, Port) ->
+    {ok, IpAddr} = inet:parse_address(Ip),
```

Comment:
We have to check how this works for Unix sockets

Reply:
Yes, this will trip over Unix sockets.

```
(VerneMQ@127.0.0.1)1> vmq_ranch_config:listeners().
[{mqtt,"/tmp/my.sock","0",running,[],10000},
 {mqtt,"127.0.0.1","1883",running,[],10000},
 {mqtts,"127.0.0.1","8883",running,[],10000},
 {vmq,"0.0.0.0","44053",running,[],10000},
 {http,"127.0.0.1","8888",running,[],10000}]
```
so we need to adapt for a call like:
```
ranch:info({{local, "/tmp/my.sock"}, 0}).
```
We should adapt `vmq_ranch_config:listener/0` to not strip the 'local' for easier matching. We might also get the `active_connection` count already in the function there, as we do for `max_connections`.